content/pt-br/docs/concepts/workloads/pods/init-containers.md

```diff
@@ -0,0 +1,266 @@
+---
+reviewers:
+- erictune
+title: Init Contêineres
+content_type: concept
+weight: 40
+---
+
+<!-- overview -->
+
+Essa página provê uma visão geral de init contêineres: especializada nos contêineres que executam antes de app contêineres em um {{< glossary_tooltip text="Pod" term_id="pod" >}}. Init contêineres podem conter utilidades ou configurações que não estão presentes em uma imagem de um app.
+
+Você pode especificar init contêineres em uma especificação de Pod ao longo de um array de `containers` (que descreve app contêineres).
```

Comment:
"Ao longo" sounds strange to me. Perhaps "juntamente com a lista de `containers`"?